python/doc/source/streams.rst

```diff
@@ -44,15 +42,10 @@ Operators use Write Streams to send data.
 Receiving Messages
 ------------------
 
-Operators receive data by registering callbacks or manually reading messages
-from Read Streams.
-
-Callbacks are functions which take an ERDOS message and any necessary write
-streams as arguments. Generally, callbacks process received messages and
-publish the results on write streams.
+Operators receive data by reading messages from Read Streams.
```

Comment:
Could you link to the ReadStream class?

Reply:
Operators also receive data by implementing callbacks that are automatically invoked upon the receipt of a message.